src/screens/Customers/EditCustomerMeter.tsx

```diff
@@ -46,53 +53,54 @@ function EditCustomerMeter() {
 
     const handleSubmit = (values: any) => {
         const { meterNumber, meterType } = values;
-        history.push('tariff-plans', { meterNumber, meterType, goBack: -2 })
+        const meterTypeKey: string = revMeterTypeMap.get(meterType) as string;
+        history.push('tariff-plans', { meterNumber, meterTypeKey, goBack: -2 })
```

Comment:
the `tariff-plans` route expects a `meterType` prop, so we need to do `history.push('tariff-plans', { meterNumber, meterType: meterTypeKey, goBack: -2 })`